views/dashboard/settings.php

```diff
@@ -1,82 +1,91 @@
 <div class="wrap">
```

Comment:
all file is wrong formatted, we use tabs and you replaced tabs with spaces instead